systemtest/src/main/java/io/strimzi/systemtest/utils/VerificationUtils.java

```diff
@@ -0,0 +1,463 @@
+     * @param podNamePrefix Prefix of pod name where container is located
+     * @param containerName The container where verifying is expected
+     * @param initialDelaySeconds expected value for property initialDelaySeconds
+     * @param timeoutSeconds expected value for property timeoutSeconds
+     * @param periodSeconds expected value for property periodSeconds
+     * @param successThreshold expected value for property successThreshold
+     * @param failureThreshold expected value for property failureThreshold
+     */
+    public static void verifyReadinessAndLivenessProbes(String namespaceName, String podNamePrefix, String containerName, int initialDelaySeconds, int timeoutSeconds, int periodSeconds, int successThreshold, int failureThreshold) {
+        LOGGER.info("Getting Pods by prefix: {} in Pod name", podNamePrefix);
+        List<Pod> pods = kubeClient(namespaceName).listPodsByPrefixInName(podNamePrefix);
+
+        if (pods.size() != 0) {
+            LOGGER.info("Verifying Readiness and Liveness configuration for container {}", containerName);
+
+            List<Container> containerList = pods.stream()
+                .flatMap(p -> p.getSpec().getContainers().stream())
+                .filter(c -> c.getName().equals(containerName))
+                .toList();
+
+            containerList.forEach(container -> {
+                // Initial delay
+                assertThat(container.getLivenessProbe().getInitialDelaySeconds(), is(initialDelaySeconds));
+                assertThat(container.getReadinessProbe().getInitialDelaySeconds(), is(initialDelaySeconds));
+                // Timeout
+                assertThat(container.getLivenessProbe().getTimeoutSeconds(), is(timeoutSeconds));
+                assertThat(container.getReadinessProbe().getTimeoutSeconds(), is(timeoutSeconds));
+                // Period
+                assertThat(container.getLivenessProbe().getPeriodSeconds(), is(periodSeconds));
+                assertThat(container.getReadinessProbe().getPeriodSeconds(), is(periodSeconds));
+                // Success threshold
+                assertThat(container.getLivenessProbe().getSuccessThreshold(), is(successThreshold));
+                assertThat(container.getReadinessProbe().getSuccessThreshold(), is(successThreshold));
+                // Failure threshold
+                assertThat(container.getLivenessProbe().getFailureThreshold(), is(failureThreshold));
+                assertThat(container.getReadinessProbe().getFailureThreshold(), is(failureThreshold));
+            });
+        } else {
+            fail("Pod with prefix " + podNamePrefix + " in name, not found");
+        }
+    }
+
+    /**
+     * Verifies metadata labels for pods
+     * @param namespaceName Namespace name where pods are located
+     * @param clusterName Name of the cluster linked with pods - used as name label prefix
+     * @param podType Name of the cluster - used as suffix with cluster name to form name label
```

Comment:
That's not name of the cluster, or?
I guess that it should for a component name (like `my-cluster-kafka`)
I would pass here the component name instead of "Pod type"

Reply:
no it is used like this: 
clusterName = CLUSTER_NAME_PREFIX + hashStub(String.valueOf(RANDOM.nextInt(Integer.MAX_VALUE)));
podType = "connect"
so its `my-cluster-0fcdd5-connect` which is only name of the pod

but i agree with renaming the pod type to component name as it suits it better, what do you think @im-konge .

Reply:
well that's exactly how it is done everywhere else
-> `KafkaConnectResources.componentName(clusterName)`
for Kafka it was `KafkaResources.kafkaComponentName(clusterName)` etc.
it's a name of the StrimziPodSet/Deployment and we are using it everywhere -> it's just outdated here

Reply:
Cool, thanks